src/prelude/prim.mo

```diff
@@ -262,8 +262,9 @@ func principalOfActor(act : actor {}) : Principal = (prim "cast" : (actor {}) ->
 // Untyped dynamic actor creation from blobs
 let createActor : (wasm : Blob, argument : Blob) -> async Principal = @create_actor_helper;
 
-// An async function for querying stable variable statistics
-let stableVarInfo = @stable_var_info;
+// Returns a query that computes the current actor's stable variable statistics (for now, the current size, in bytes, of serialized stable variable data).
+func stableVarQuery() : shared query () -> async {size : Nat64} =
+  (prim "stableVarQuery" : () -> (shared query () -> async {size : Nat64})) () ;
```

Comment:
Oh, I see. The `prim` returns a query function. Which is a pair of a `(principal,〈method-name〉)`.

Reply:
Yep. I may actually be able to get rid of the clunky `() ->...` delay if I shove this into internals and reference it. I don't think we do a definedness check there ;->